angular-app/src/app/app.module.ts

```diff
@@ -81,7 +85,13 @@ import { ToggleChatService } from './toggle-chat.service';
   ],
   providers: [
     AuthService,
-    ToggleChatService
+    ToggleChatService,
+    {
+      provide: MAT_CHIPS_DEFAULT_OPTIONS,
+      useValue: {
+        separatorKeyCodes: [ ENTER, COMMA ]
```

Comment:
Do you expect to have spaces in your tags? If so, nevermind. Otherwise, space might be a good candidate to add here.